omglol/paste.go

```diff
@@ -0,0 +1,166 @@
+package omglol
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+// Create a Paste object
+func NewPaste(Title, Content string, ModifiedOn int64) *Paste {
+	return &Paste{
+		Title:      Title,
+		Content:    Content,
+		ModifiedOn: ModifiedOn,
```

Comment:
`ModifiedOn` will be added by the API, so make it optional when used for `CreatePaste`. Also add the `listed` attribute.
```suggestion
	func NewPaste(Title, Content string, Listed bool, ModifiedOn ...int64) *Paste {
	var modified *int64
	if len(ModifiedOn) > 0 {
		m := ModifiedOn[0]
		modified = &m
	}
	
	return &Paste{
		Title:      Title,
		Content:    Content,
		Listed:     Listed,
		ModifiedOn: modified,
```